### PR TITLE
show word list and word of category

### PR DESCRIPTION
**Pull request by thanhbk55**

### CODE REVIEW

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class CategoriesController < ApplicationController
   def index
     @categories = Category.all.paginate page: params[:page], per_page: 5
+    @words = Word.all.paginate page: params[:page], per_page: 5
```
Comment (ThuBM) [inline app/controllers/categories_controller.rb]: remove .all

File: app/controllers/words_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class WordsController < ApplicationController
+  def index
+    @categories = Category.all.paginate page: params[:page], per_page: 5
```
Comment (ThuBM) [inline app/controllers/words_controller.rb]: remove .all

File: app/controllers/categories_controller.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class CategoriesController < ApplicationController
   def index
     @categories = Category.all.paginate page: params[:page], per_page: 5
```
Comment (ThuBM) [inline app/controllers/categories_controller.rb]: remove .all

File: app/views/categories/index.html.erb [hunk 4]
```diff
@@ -5,4 +5,16 @@
...
-</div>
\ No newline at end of file
+</div>
+<% @categories.each do |category| %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title"><%= category.name %></h3>
+    </div>
+   <div class="panel-body">
+       <div class="categories-word-<%= category.id %>">
```
Comment (ThuBM) [inline app/views/categories/index.html.erb]: indent

File: app/views/categories/index.html.erb [hunk 5]
```diff
@@ -5,4 +5,16 @@
...
+        	<%= render category.words %>
```
Comment (ThuBM) [inline app/views/categories/index.html.erb]: indent

File: app/views/words/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<% @categories.each do |category| %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title"><%= category.name %></h3>
+    </div>
+   <div class="panel-body">
+       <div class="categories-word-<%= category.id %>">
```
Comment (ThuBM) [inline app/views/words/index.html.erb]: indent

File: app/views/words/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+        	<%= render category.words %>
```
Comment (ThuBM) [inline app/views/words/index.html.erb]: indent

File: app/views/categories/index.html.erb [hunk 8]
```diff
@@ -5,4 +5,16 @@
...
+        </div>
+    </div>
+</div>
```
Comment (yuyuvn) [inline app/views/categories/index.html.erb]: indent

File: app/views/static_pages/home.html.erb [hunk 9]
```diff
@@ -6,6 +6,9 @@
         <%= render "shared/stats" %>
       </section>
     </aside>
+    <div class="learn-form">
+       <%= render "shared/learn_form" %>
```
Comment (yuyuvn) [inline app/views/static_pages/home.html.erb]: indent

File: app/views/words/_word.html.erb [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<li id="word-<%= word.id %>">
+   <span class="word"><%= word.content %></span>
```
Comment (yuyuvn) [inline app/views/words/_word.html.erb]: indent

File: app/views/words/index.html.erb [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+        </div>
+    </div>
```
Comment (yuyuvn) [inline app/views/words/index.html.erb]: indent

File: config/routes.rb [hunk 12]
```diff
@@ -5,8 +5,10 @@
   get "login" => "sessions#new"
   post "login" => "sessions#create"
   delete "logout" => "sessions#destroy"
-
+  get "category" => "categories#index"
+  get "wordlist" => "words#index"
```
Comment (yuyuvn) [inline config/routes.rb]: use restful

File: config/routes.rb [hunk 13]
```diff
@@ -5,8 +5,8 @@
   get "login" => "sessions#new"
   post "login" => "sessions#create"
   delete "logout" => "sessions#destroy"
-
   resources :categories
+  resources :words
```
Comment (ThuBM) [inline config/routes.rb]: only

**Comment by ThuBM** [comment]
OK

**Pull request merged**